pkg/xds/bootstrap/template_v3.go

```diff
@@ -158,10 +159,12 @@ func genConfig(parameters configParameters, proxyConfig xds.Proxy, enableReloada
 		DynamicResources: &envoy_bootstrap_v3.Bootstrap_DynamicResources{
 			LdsConfig: &envoy_core_v3.ConfigSource{
 				ConfigSourceSpecifier: &envoy_core_v3.ConfigSource_Ads{Ads: &envoy_core_v3.AggregatedConfigSource{}},
+				InitialFetchTimeout:   durationpb.New(0),
 				ResourceApiVersion:    envoy_core_v3.ApiVersion_V3,
 			},
 			CdsConfig: &envoy_core_v3.ConfigSource{
 				ConfigSourceSpecifier: &envoy_core_v3.ConfigSource_Ads{Ads: &envoy_core_v3.AggregatedConfigSource{}},
+				InitialFetchTimeout:   durationpb.New(0),
```

Comment:
Sanity check: is this also going to apply for ClusterLoadAssignment?

Reply:
ClusterLoadAssignment is covered in EDS, located at `pkg/xds/envoy/clusters/v3/eds_cluster_configurer.go`

Reply:
Yeah we need to set that as well I think